test/dice_test.coffee

```diff
@@ -0,0 +1,27 @@
+Helper = require('../node_modules/hubot-test-helper/src/index.coffee')
+
+helper = new Helper('../scripts/dice.coffee')
+
+expect = require('chai').expect
```

Comment:
Can you make a case why this is worth using over `assert`? I'd rather keep the dependency list as small as we can manage.